### ISSUE TITLE
Axe throwing error for role="figure"

### ISSUE THREAD

**Issue by scottaohara** [dequelabs/axe-core#1460]
## Expectation:
Not all browser and screen reader combinations correctly expose the `figure` element and its caption.  Using `role="figure"` can help alleviate this gap for the time being.

People should be able to use `role="figure"` without receiving an error.

## Actual: 
axe (and lighthouse) return "ARIA roles used must conform to valid values" when using a `role="figure"` on an element.

e.g.
```html
<div role="figure" aria-label="text goes here"> 
  <img src="my-image.jpg" alt="alt text here"> 
  <p>text goes here.</p> 
</div>
```

## Motivation: 
The rule should be corrected because [`role="figure"` is a valid role](https://www.w3.org/TR/wai-aria-1.1/#figure).

<pre><code>axe-core version: 3.2.2
axe-devtools 3.8.0
</code></pre>

**Comment by WilcoFiers** [comment]
@scottaohara Do you have test information about how your example works in things like VO+Safari, NVDA+FF and JAWS+IE? If this solution does indeed work in those, I fully agree that this needs to be changed in axe.

**Comment by scottaohara** [comment]
As a matter of fact :) 
https://www.scottohara.me/blog/2019/01/21/how-do-you-figure.html 
where I talk about the support for `<figure>` (and `role=figure`) quite a bit.  tldr; `role=figure` is necessary to expose figures in IE11+JAWS, and additionally is exposed correctly when testing with VO+Safari.

Additionally I have the following demo: https://scottaohara.github.io/testing/automated/ for a simple test case.

I've opened an issue with NVDA regarding [`figure` and `role=figure`](https://github.com/nvaccess/nvda/issues/9177) and their lack of communicating the element/role.

With that said, I'd still find it odd to flag it as an "invalid role" when it's not an invalid role. 

A best practice or warning of "this role may not be supported in all browsers/AT" sure, I get that.  But if you take that stance with `role=figure` then why does the `<meter>` element, for example, not cause any flags?  It has abysmal accessibility support.

**Comment by chandana7393** [comment]
Axe-coconut is not throwing error for role="figure"
![figure_role](https://user-images.githubusercontent.com/45285854/58697601-677cd980-83b7-11e9-873d-76e4c305d754.PNG)
**Tested Environment:**
Axe-coconut - 3.8.0.15004v
Chrome - 74v
OS - Windows 10 64 bit.
